images/linux/scripts/installers/dotnetcore-sdk.sh

```diff
@@ -67,4 +68,10 @@ setEtcEnvironmentVariable DOTNET_NOLOGO 1
 setEtcEnvironmentVariable DOTNET_MULTILEVEL_LOOKUP 0
 prependEtcEnvironmentPath '$HOME/.dotnet/tools'
```

Comment:
@al-cheb was looking at the script again and just noticed this line here...
seems that this is already being added to the path, maybe not correct? or misplaced? or pointing to the wrong path?